.circleci/config.yml

```diff
@@ -873,9 +892,15 @@ workflows:
             - tornado
             - unit_tests
             - vertica
-      - deploy_dev:
+      - deploy_master:
           requires:
             - verify_all_tests_ran
           filters:
             branches:
               only: master
```

Comment:
if you want to test these out, can push with:

```
only:
  - master
  - your-feature-branch
```

then can verify they work as expected before we merge them

Reply:
I've tested locally and it works — I just didn't test the upload itself since I don't have the creds. :)

Reply:
Just pushed to `build-staging-branch` to see if it works on CircleCI.

Reply:
I see success https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/206/workflows/19d60f14-c7d0-455b-b943-5ff6ff3d8063/jobs/203800